plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -79,8 +79,10 @@ class ServiceNetworkingConfigSection extends ServiceConfigBaseSectionDisplay {
                 prop: keys.protocol,
                 className: getColumnClassNameFn(),
                 render(prop, row) {
-                  return getDisplayValue(row[prop])
-                    .split(',').join(', ');
+                  let protocol = row[prop] || '';
+                  protocol = protocol.replace(',', ', ');
```

Comment:
Currently it is an `enum` value can be `udp`, `tcp` or `tcp,udp` and that guy `/,.*/g` will just eat the `,udp` part entirely. So we could go with `//g` flag but I wouldn't change the expression itself.